fidesctl/src/fideslang/models.py

```diff
@@ -123,6 +123,9 @@ class DataQualifier(FidesModel):
 class DataSubject(FidesModel):
     """The DataSubject resource model."""
 
+    rights_available: Optional[str]
```

Comment:
This should likely be a list of strings at minimum to allow for multiple rights. Originally there may have been a want to have a separate taxonomy or enum here and it seems to make sense going forward (as they are defined in [chapter 3 of the GDPR](https://gdpr-info.eu/chapter-3/))

Reply:
Circling back to the original issue #369, a preference here seemed to apply all as a default and only annotate the rights not available (feedback made it sound like that may be the best user experience)

The next question would be to list all of change the output of the data map to only reflect rights not available to the data subject 🤔